bin/install-qa-check.d/60openrc

```diff
@@ -33,6 +33,23 @@ openrc_check() {
 			done
 		done
 	fi
+
+	# Make sure deprecated #!/sbin/runscript isn't being used:
+	if [[ ! -d "${ED}"/etc/init.d ]] ; then
+		continue
```

Comment:
It looks like you meant use to use "return" instead of "continue" here? I think it's nicer to just nest your other code inside like:
```
if [[ -d ${ED}/etc/init.d ]] ; then
    shopt -s nullglob
    for x in "${ED}"/etc/init.d/* ; do
    ...
fi
```